hack/update-generated-conversions.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to fetch latest swagger spec.
+# Puts the updated spec at swagger-spec/
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function generate_version() {
+	local version=$1
+
+	echo "Generating for version ${version}"
+
+	sed 's/YEAR/2015/' hooks/boilerplate.go.txt > /tmp/conversion_generated.go
+	cat >> /tmp/conversion_generated.go <<EOF
```

Comment:
You should make a tmp variable at the top of teh file and use it everywhere.  

TMPFILE="/tmp/conversion_generated.$(date +%s).go"